docs/concepts/storage/persistent-volumes.md

```diff
@@ -494,7 +492,7 @@ spec:
 
 ### Binding Block Volumes
 
-If a user requests a raw block volume by indicating this using the volumeMode field in the PersistentVolumeClaim spec, the binding rules differ slighty from previous releases that didn't consider this mode as part of the spec.
+If a user requests a raw block volume through the persistentVolumeClaim.volumeMode field, it can only bind to PersistentVolume with the matching volumeMode field.
 Listed is a table of possible combinations the user and admin might specify for requesting a raw block device. The table indicates if the volume will be bound or not given the combinations:
 Volume binding matrix for statically provisioned volumes:
```

Comment:
I think the matrix is a little outdated now.  Because we default the volumeMode, there is no such thing as "unspecified" anymore.  Once you remove unspecified from the matrix, it really does simplify to just the modes on pv and pvc have to be the same.

Reply:
@erinboyd 

Agree on above.
The matrix contains actual values can be set to the volumeMode and condition of the volumeMode.
- Actual values to set volumeMode: Block or Filesystem
- Condition of the parameter: unspecified

We can remove the matrix then explain simply as @msau42 mentioned.